canary_tests.py

```diff
@@ -35,11 +35,7 @@ async def connect(address, opts):
 
```

Comment:
While we're tinkering with this file, change line 32 to use `address` instead of `conf.address`, so it matches line 19.